isis/pytests/test_IsisDownload.py

```diff
@@ -59,3 +60,33 @@ def test_create_rclone_args():
         dest = Path(tdir)
         args = did.create_rclone_arguments(str(dest), "lro_naifKernels:", ntransfers=100, rclone_kwargs=["--dry_run"])
         assert args == ['lro_naifKernels:', str(dest/"lro"/"kernels"), '--progress', '--checkers=100', '--transfers=100', '--track-renames', '--log-level=WARNING', '--dry_run']
+
+
+def test_file_filtering():
+    files_to_exclude = [
+        "/a_older_versions/*",
+        "/kernels/*/former_versions/",
+        "/corrupt_files/*",
+        "/zzarchive/**",
+        "/kernels/*/original",
+        "gallileo/kernels/ck/prime_mission/*/extended_mission/*",
+        "gallileo/kernels/*/prime_mission/",
+        "galileo/kernels/ck/GEM/",
+        "clementine1/kernels/ck/save",
+        "chandrayaan1/kernels/spk/SAVE_SCS_2017-11-22",
+        "*/kernels/fk/release.*/*",
+        "cassini/kernels/fk/Archive"
+    ]
```

Comment:
If you are sourcing the script, you can probably use the same exclude list from the actual script rather than duplicating here.